### DISCUSSION TITLE
Where is the module/script api/token_config.py found and what should it contain?

### DISCUSSION THREAD

**Discussion by pajot** [openego/oeplatform#930] [Q&A] [answered]
[In this tutorial](https://openenergy-platform.org/tutorials/jupyter/OEP_API_tutorial_part1/#0.-Setup-token), there is a reference to a module named `token_config`, but it is not otherwise explained.

[Here, a file api/token_config.py is referenced.](https://openenergy-platform.org/tutorials/jupyter/OEP_API_tutorial_part1/#3b.--Paste-your-token-in-a-file-(discouraged)) Neither does that file exist, nor is there a complete explanation of how to create it and what it should contain.

**Comment by jh-RLI** [comment, marked as answer]
Good hint, this is not well documented. Mainly this is because we develop the JuptyNotebook based tutorials in a [separate Github repository](https://github.com/OpenEnergyPlatform/tutorial). There is the module [token_config.py](https://github.com/OpenEnergyPlatform/tutorial/blob/develop/api/token_config.py) . I will create the question in the tutorial repo. as an issue. 
We recommend there to store the OEP API token in the local environment as an environment variable.

**Reply by pajot** [reply]
Thanks for the tip, I didn't think to look in the tutorial repo.
